test_files/reexport.declaration/reexport.ts

```diff
@@ -0,0 +1 @@
+export * from './export';
```

Comment:
Can you add some comments to these test files about what they're testing, like what property of the output is important?

Reply:
Added a comment.